tests/test_chatmember.py

```diff
@@ -96,6 +98,7 @@ def chat_member_restricted():
         CMDefaults.can_change_info,
         CMDefaults.can_invite_users,
         CMDefaults.can_pin_messages,
+        CMDefaults.can_manage_topics,
```

Comment:
this is not in the correct position. Should be after line 106, or line 107 depending on the review comment above